src/Visualization.jl

```diff
@@ -152,3 +152,36 @@ function Visualization.create_vtk_file(
       celldata=c,nodaldata=n)
   end
 end
+
+struct DistributedPvd{T}
+  pvds::AbstractPData{T}
```

Comment:
This struct is not type-stable. Simply define it like this:

```julia
struct DistributedPvd{T<:AbstractPData}
  pvds::T
end
```

Reply:
fixed in https://github.com/oriolcg/GridapDistributed.jl/commit/a0b6f85e434d51ced94c61b1b2033c6960aee742